cypress/integration/challenge-tests/updateTests.js

```diff
@@ -0,0 +1,8 @@
+// /* global cy */
+
+// Here you can introduce a new superblock
+// After this run cypress:dev:watch and new specfiles will be generated automatically
+
+//  it('should update', () => {
+//     cy.updatePaths('scientific-computing-with-python')
+//  })
```

Comment:
I think this file can go.

Reply:
Hmm, thought I deleted that in the last commit, oops.